new-website/src/components/location/edit-form.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react"
+import { Location } from '@reach/router'
+import { makeStyles } from '@material-ui/core/styles';
+import { TextField, Button } from '@material-ui/core';
+import firebase from "gatsby-plugin-firebase"
+import { navigate, useIntl } from "gatsby-plugin-intl"
+import SaveIcon from '@material-ui/icons/Save';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+export const EditForm = ({ partnerId, collection, docId, doc, onError }) => {
+  const classes = useStyles()
+  const { formatMessage } = useIntl()
+
+  const [title, setTitle] = useState(doc.title || "")
```

Comment:
I have been using react-use-form just for convinience, simplicity, performance and I'm plain lazy ... see login.jsx
https://react-hook-form.com/

Reply:
Followed your example and had a look at some of the docs, but didn't get it to work: https://github.com/weliveindetail/ImmunHelden.de/commit/82b60d3ec2932c8169553a817b62b0808c5d60d9

Reply:
What was the error you had... I mean we can merge it without it and add it later as well

Reply:
No error, just not getting back any values: https://github.com/weliveindetail/ImmunHelden.de/commit/82b60d3ec2932c8169553a817b62b0808c5d60d9#diff-70a4ac7d4b1a5bb8053ad94f7f0bc3b6R28

Anyway, I gave up on it nad I think it's fine.